src/charm.py

```diff
@@ -435,7 +471,7 @@ def _all_config_or_relations(self) -> Any:
     @property
     def _multiple_relations(self) -> bool:
         """Return a boolean indicating if we're related to multiple applications."""
-        return len(self.model.relations["ingress"]) > 1
+        return len(self.model.relations["ingress"] + self.model.relations["nginx-route"]) > 1
```

Comment:
I think this creates a new list, what about `len(self.model.relations["ingress"]) + len(self.model.relations["nginx-route"])`

Reply:
Fixed, thanks!